lib/xcsoar/mapgen/waypoints/seeyou_reader.py

```diff
@@ -76,26 +76,28 @@ def __parse_length(str):
 def parse_seeyou_waypoints(lines, bounds=None):
     waypoint_list = WaypointList()
 
-    first = True
-    for line in lines:
-        if first:
-            first = False
-            continue
+    header = "name,code,country,lat,lon,elev,style,rwdir,rwlen,freq,desc"
```

Comment:
beware that the there is a new standard since 2022, that has some additional fields:
https://github.com/naviter/seeyou_file_formats/blob/main/CUP_file_format.md

Maybe xcsoar's parser can serve as a reference:
https://github.com/XCSoar/XCSoar/blob/master/src/Waypoint/WaypointReaderSeeYou.cpp